_episodes/02-installation.md

```diff
@@ -40,6 +40,10 @@ documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/install
     should work on any recent system. If you have problems in the next step(s)
     you can alternatively try a 32 bit version.
```

Comment:
Please replace all of the text before the code block below with "Please download Miniconda3 by running"